zebra-rpc/src/methods/tests/snapshot.rs

```diff
@@ -0,0 +1,242 @@
+//! RPC responses
+
+use insta::dynamic_redaction;
+use std::sync::Arc;
+
+use zebra_chain::{
+    block::Block,
+    parameters::Network::{Mainnet, Testnet},
+    serialization::ZcashDeserializeInto,
+};
+
+use zebra_network::constants::USER_AGENT;
+use zebra_node_services::BoxError;
+use zebra_test::mock_service::MockService;
+
+use super::super::*;
+
+/// Snapshot test for RPC methods responses.
+///
+/// TODO:
+/// - Add a `z_gettreestate` test when #3990 is merged.
```

Comment:
Missing tests:
- Can you please open a ticket for the `z_gettreestate` snapshot?
- Did you want to do a `sendrawtransaction` snapshot in this PR, or in another ticket?

Reply:
For `z_gettreestate` opened ticket https://github.com/ZcashFoundation/zebra/issues/4379